odoo/addons/base/models/ir_http.py

```diff
@@ -270,6 +271,9 @@ def routing_map(self, key=None):
 
     @api.autovacuum
     def _gc_sessions(self):
+        if not config["db_name"]:
```

Comment:
So something like so?
```suggestion
        if os.getenv('ODOO_SKIP_GC_SESSIONS'):
```